apis/druid/v1alpha1/druid_types.go

```diff
@@ -24,6 +24,22 @@ import (
 // EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
+// AdditionalContainer defines the additional sidecar container
+type AdditionalContainer struct {
+	// List of configurations to use which are not present or to override default implementation configurations
+	// +optional
+	//Configs                  map[string]string `json:"configs,omitempty"`
+	Image 					 string `json:"image,omitempty"`
+	ContainerName			 string `json:"containerName,omitempty"`
+	Command					 []string `json:"command,omitempty"`
+	ImagePullPolicy 		 v1.PullPolicy `json:"imagePullPolicy,omitempty"`
+	Args                     []string                `json:"args,omitempty"`
+	ContainerSecurityContext *v1.SecurityContext     `json:"securityContext,omitempty"`
+	Resources                v1.ResourceRequirements `json:"resources,omitempty"`
+	VolumeMounts         	 []v1.VolumeMount           `json:"volumeMounts,omitempty"`
+	Env 					 []v1.EnvVar `json:"env,omitempty"`
+}
```

Comment:
can we format this and remove //configs as a comment

Reply:
Sure. Will do that